Content.Shared/_Umbra/Preferences/Loadouts/Effects/PersonalItemLoadoutEffect.cs

```diff
@@ -29,7 +31,17 @@ public override bool Validate(
         IDependencyCollection collection,
         [NotNullWhen(false)] out FormattedMessage? reason)
     {
-        if (!CharacterName.Contains(profile.Name))
+        var foundMatchingName = false;
+        foreach (var name in CharacterName)
+        {
+            if (name.Equals(profile.Name, StringComparison.OrdinalIgnoreCase))
+            {
+                foundMatchingName = true;
+                break;
+            }
+        }
```

Comment:
I hate this, but `Hashset<String>.contains(element, comparitor)` doesn't work because the `StringComparer` type isn't whitelisted in Robust Toolbox :)